setup/install/deploy.md

```diff
@@ -52,6 +53,15 @@ deploy apply`.
 
 1. Navigate to [localhost:9000](localhost:9000).
 
+
+__Note:__ Even if the `hal deploy apply` command returns successfully, the 
+installation may not be complete. This is especially the case with 
```

Comment:
```suggestion
installation may not be complete yet. This is especially the case with 
```

Reply:
My thinking here is that adding this one word makes it clear that it's a waiting thing, rather than something just didn't finish.